downstream/modules/devtools/proc-rhdh-create-plugin-registry.adoc

```diff
@@ -0,0 +1,47 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="rhdh-create-plugin-registry_{context}"]
+= Creating a plug-in registry
+
+Set up a plug-in registry in your OpenShift cluster to host the Ansible plug-ins and make them available for installation in Red Hat Developer Hub.
+
+.Procedure
+
+. Log in to your {OCPShort} instance with credentials to create a new application.
+. Open your {RHDH} OpenShift project.
++
+----
+$ oc project <YOUR_DEVELOPER_HUB_PROJECT>
+----
+. Run the following commands to create a plug-in registry build in the OpenShift cluster.
++
+----
+$ oc new-build httpd --name=plugin-registry --binary
+$ oc start-build plugin-registry --from-dir=$DYNAMIC_PLUGIN_ROOT_DIR --wait
+$ oc new-app --image-stream=plugin-registry
+----
+
+.Verification
+
+To verify that the plugin-registry was deployed successfully, open the Developer perspective on the {RHDH} application in the OpenShift Web console.
+
+. Click the plug-in registry to view the log. 
++
+image::rhdh-plugin-registry.png[Developer perspective]
```

Comment:
This image doesn't show in your preview PR, but that might expect behaviour for a pr-preview?